src/main/resources/validationrules/civil/civil-admin-mapping.drl

```diff
@@ -418,3 +421,38 @@ then
       .build());
       logMsg("Rule : civil_case_allocator_admin_org_role");
 end;
+
+/*
+ * CIVIL admin and hearing centre team leader "case-allocator" Org role mapping.
+ */
+
+rule "civil_case_allocator_admin_hearing_centre_team_leader_org_role"
```

Comment:
Can we rename rule **and update log entry** to: `v2_1_civil_case_allocator_admin_org_role`.

Reply:
Changes made as requested.

Reply:
Resolved in 80c7475aff3163939e73440c328bdf27bcdf563a.